homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -96,6 +101,7 @@ async def async_step_prompt_options(self, user_input=None):
         if user_input is not None:
             self._global_options = {
                 CONF_AUTOMATIC_ADD: user_input[CONF_AUTOMATIC_ADD],
+                CONF_PROTOCOLS: user_input.get(CONF_PROTOCOLS) or None,
```

Comment:
user input will always contain the key. No need for get()

Reply:
Thanks. This is my first foray into HA development, learning voluptuous and lovelace has been fun, it's a very slick framework.